packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -146,39 +147,39 @@ describe('anchor', () => {
 
     test('enforces schema in update that assigns schema', async () => {
       const schemaDoc = await TileDocument.create(ceramic, STRING_MAP_SCHEMA)
-      await TestUtils.anchorUpdate(ceramic, schemaDoc)
+      await anchorUpdate(ceramic, schemaDoc)
 
       const stream = await TileDocument.create(ceramic, { stuff: 1 })
       const streamState = await ceramic.repository.load(stream.id, {})
-      await TestUtils.anchorUpdate(ceramic, stream)
-      const updateRec = await stream.makeCommit(ceramic, null, { schema: schemaDoc.commitId })
+      await anchorUpdate(ceramic, stream)
+      const updateRec = await stream.makeCommit(ceramic.signer(), null, {
```

Comment:
ditto my comment above about `TileDocument.makeGenesis`.  The fact that this change here is needed is a sign that we've made a breaking change to our public API.